ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -216,6 +260,54 @@ export default class StatusDropdown extends React.Component {
     alertStatus === 'investigating' ||
     (alertStatus !== status && this.isResolved(alertStatus));
 
+  calculateDueDate(created) {
+    const createdAt = new Date(created);
+    const dueDate = new Date(created);
+    dueDate.setDate(dueDate.getDate() + 3);
+    const numberOfNonWorkingDays = this.getNumberOfNonWorkingDays(
+      createdAt,
+      dueDate,
+    );
+
+    dueDate.setDate(
+      numberOfNonWorkingDays !== 0
+        ? dueDate.getDate() + numberOfNonWorkingDays
+        : dueDate.getDate(),
+    );
+
+    return dueDate;
+  }
+
+  renderDueDateCountdown(createdAt) {
+    const now = new Date(Date.now());
+    const dueDate = this.calculateDueDate(createdAt);
+
+    // const created = new Date(createdAt);
+
+    // console.log('---------');
+    // console.log(created, 'day created');
+    // console.log(now, 'date now');
+    // console.log(dueDate, 'dueDate');
+    // console.log('---------');
+
+    const diffTime = Math.abs(dueDate - now);
+    const day = Math.ceil(diffTime / (1000 * 60 * 60 * 24));
+
+    if (now.getDate() === dueDate.getDate()) {
+      return 'Today';
+    }
+
+    if (now.getTime() >= dueDate.getTime()) {
+      return 'Overdue';
+    }
+
+    if (this.isWeekend) {
+      return `${day - 2} working days`;
```

Comment:
It would be nice to display the message '1 working day' if there's only one day left, not '1 working days'.